src/components/mote/components/innkalling/InnkallingDialogmotePanel.tsx

```diff
@@ -10,16 +10,36 @@ import { Moteplanleggeren } from "./Moteplanleggeren";
 import { DialogmoteMoteStatusPanel } from "./DialogmoteMoteStatusPanel";
 import { useAktivtDialogmote } from "../../../../data/dialogmote/dialogmote_hooks";
 import { useIsDM2Enabled } from "../../../../data/unleash/unleash_hooks";
+import { useNavBrukerData } from "../../../../data/navbruker/navbruker_hooks";
+import styled from "styled-components";
+import { AlertstripeFullbredde } from "../../../AlertstripeFullbredde";
+import { Normaltekst } from "nav-frontend-typografi";
 
-const texts = {
+export const texts = {
   bekreftetMote: "Bekreftet møte",
   seMotestatus: "Se møtestatus",
   planleggNyttMote: "Planlegg nytt dialogmøte",
   ingenMoterPlanlagt: "Ingen møter planlagt",
   dialogMote: "Dialogmøte",
   moteforesporselSendt: "Møteforespørsel sendt",
+  kanIkkeVarslesPart1:
+    "Innbyggeren er reservert mot elektronisk kommunikasjon med det offentlige, eller har ingen kontaktinformasjon registrert, vi kan derfor ikke sende varsler til personen.",
```

Comment:
Nitpick: Tenker veileder leser setningen litt lettere dersom du har punktum etter registrert, i stedet for komma.